src/ProxyClient/Varnish.php

```diff
@@ -151,6 +160,8 @@ protected function configureOptions()
         $resolver->setDefaults([
             'tags_header' => self::DEFAULT_HTTP_HEADER_CACHE_TAGS,
             'header_length' => 7500,
+            'tags_invalidation_by_purge' => false,
+            'tags_separator' => ',',
```

Comment:
Xkey tag header format is list of tags separated by space, while the one used for ban now is comma separated, right?

That said, the other approach I had in mind was to change defaults to use xkey, and adapt BAN VCL and Symfony proxy store to understand that natively _(to simplify learning use of Tagging by making it consistent across approaches). But _that_ is a bc break, so a bit late for 2.0.